Code/Framework/AzGameFramework/AzGameFramework/Application/GameApplication.cpp

```diff
@@ -72,15 +72,25 @@ namespace AzGameFramework
 
         MergeSharedSettings(registry, specializations, scratchBuffer);
 
-        // Used the lowercase the platform name since the bootstrap.game.<config>.setreg is being loaded
-        // from the asset cache root where all the files are in lowercased from regardless of the filesystem case-sensitivity
-        static constexpr char filename[] = "bootstrap.game." AZ_BUILD_CONFIGURATION_TYPE  ".setreg";
-
-        AZ::IO::FixedMaxPath cacheRootPath;
-        if (registry.Get(cacheRootPath.Native(), AZ::SettingsRegistryMergeUtils::FilePathKey_CacheRootFolder))
+        // Query the launcher type from the registry
+        constexpr AZStd::string_view LauncherTypeTag = "/O3DE/Runtime/LauncherType";
+        using FixedValueString = AZ::SettingsRegistryInterface::FixedValueString;
+        if (FixedValueString launcherType; registry.Get(launcherType, LauncherTypeTag)
+            && !launcherType.empty())
         {
-            cacheRootPath /= filename;
-            registry.MergeSettingsFile(cacheRootPath.Native(), AZ::SettingsRegistryInterface::Format::JsonMergePatch, "", &scratchBuffer);
+            // Use the lowercase the platform name since the bootstrap.<launcher-type>.<config>.setreg is being loaded
```

Comment:
Use the lowercase platform name…

Reply:
Actually that comment is a holdover for when we used to load the `bootstrap.game.<platform>.<config>.setreg` file.
It is no longer needed